functional_tests/scylla_operator/__init__.py

```diff
@@ -12,3 +12,14 @@
 # See LICENSE for more details.
 #
 # Copyright (c) 2021 ScyllaDB
+
+
+def get_pods_services_lists_and_validate_services(db_cluster, message):
+    pod_names = db_cluster.scylla_pod_names
+    services_names = db_cluster.scylla_services_names
+    compare_and_validate(pod_names, services_names, f'Orphaned services were found {message}')
+
+
+def compare_and_validate(first, second, error_message):
+    diff = list(set(second) - set(first))
+    assert not diff, f"{error_message}: {','.join(diff)}"
```

Comment:
I think such things must be defined somewhere here:
**functional_tests/scylla_operator/libs**

Then, "get_pods_services_lists_and_validate_services" sounds very strange to me.
It describes the approach, but must describe the idea. And the idea is to find orphaned services or "mismatch" of services and pods.

Moreover, I think that better to create, if create, general function which allows to apply selectors for pod and service lists and then match them.

Reply:
Agree, also want to add that it is always better to assert in the test body, when it is possible.

It is better to have `get_orphaned_resources(db_cluster) -> List[str]`
and do :

```
assert not get_orphaned_resources()
```

in the test

Reply:
fixed